cpacs2to3/cpacs_converter.py

```diff
@@ -414,6 +415,19 @@ def convert_non_explicit_stringer(tixi3):
         tixi3.addDoubleElement(path, 'xsi', xsi, '%g')
         tixi3.addTextElement(path, 'referenceUID', uid)
 
+def rearrange_non_explicit_stringer(tixi3):
+    """schema order of angle and refpoint of wingStringerType changed.
+
+    In 3.1 angle should be after refPoint
+    """
+    for path in tixihelper.resolve_xpaths(tixi3, '//lowerShell/stringer|//upperShell/stringer|//cell/stringer'):
+        if not tixi3.checkElement(path + '/pitch'):
+            continue
+        angle = tixi3.getDoubleElement(path + '/angle')
+        tixi3.removeElement(path + '/angle')
```

Comment:
You could also use ``tixiSwapElements`` here alternatively.

Reply:
cool - implemented